front/src/modules/trainschedule/components/TimetableV2/TimetableTrainCardV2.tsx

```diff
@@ -216,9 +216,16 @@ const TimetableTrainCardV2 = ({
                 </span>
               )}
             </div>
-            <div className="scenario-timetable-train-times V2">
-              <div className="scenario-timetable-train-departure">{train.startTime}</div>
-              <div className="scenario-timetable-train-arrival">{train.arrivalTime}</div>
+            <div
+              className={cx('scenario-timetable-train-times V2', {
+                'not-honored': !train.scheduledPointsHonored,
+              })}
+            >
+              <div className="ml-1">{!train.scheduledPointsHonored && <Clock size="lg" />}</div>
```

Comment:
I think it's better if you simply not render the div, so you should test like this:
```suggestion
              {!train.scheduledPointsHonored && (
                <div className="ml-1">
                  <Clock size="lg" />
                </div>
              )}
```

Reply:
Done